Dockerfile

```diff
@@ -318,6 +318,9 @@ RUN echo "${VERSION}" > /src/qsiprep/qsiprep/VERSION && \
 RUN python -c "from matplotlib import font_manager" && \
     sed -i 's/\(backend *: \).*$/\1Agg/g' $( python -c "import matplotlib; print(matplotlib.matplotlib_fname())" )
 
+RUN mkdir -p ${HOME}/.dipy
+RUN python -c "import amico; amico.core.setup()"
```

Comment:
This is going to write to /home/qsiprep/.dipy, but Singularity users will have their own $HOME. I got around this by exporting DIPY_HOME in the Dockerfile

Reply:
Earlier in the Dockerfile it creates a unique $HOME, this turned out to be ok